src/algorithms/slp_line_search.jl

```diff
@@ -49,7 +49,7 @@ mutable struct SlpLS{T,Tv,Tt} <: AbstractSlpOptimizer
         slp.alpha = 1.0
 
         slp.options = problem.parameters
-        slp.optimizer = slp.options.external_optimizer()
+        slp.optimizer = MOI.instantiate(slp.options.external_optimizer)
```

Comment:
Does this still work if `external_optimizer::DataType`?

Reply:
Yes. `MOI.instantiate` works for both `DataType` and `MOI.OptimizerWithAttributes`